pluginWorker.js

```diff
@@ -50,7 +50,7 @@ function PluginLoader(settings) {
             data: _result
         });
     }, function (err) {
-        console.log(JSON.parse(JSON.stringify(err)));
```

Comment:
This one is a bit obtuse.  pluginWorker.js is a thread worker, and thread workers don't have a console defined.  So, `console.error` does not exist.  Take a closer look [here](https://github.com/StephenGregory/Brackets-InteractiveLinter/blob/sg/clean-up-logic/pluginWorker.js#L78). I define a custom `console.log`, which is sending the message to be logged across from the worker thread to the main execution program.  What the JSON.parse(JSON.stringify) part is doing is removing all the functions from `err`, and that's because messages sent to and from a worker thread cannot have functions in them; they must simple objects.